sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -94,7 +94,7 @@ public DefaultAndroidEventProcessor(
 
     ExecutorService executorService = Executors.newSingleThreadExecutor();
     // dont ref. to method reference, theres a bug on it
-    contextData = executorService.submit(() -> loadContextData());
+    contextData = executorService.submit(this::loadContextData);
```

Comment:
```suggestion
    contextData = executorService.submit(() -> loadContextData());
```
there's a comment above ;)

Reply:
yeah, I just noticed that too after going over it one last time. disabled the linter for the line